### PR TITLE
Add error scenarios for `ownerOf`

### PR DESCRIPTION
**Pull request by saturnial**

### CODE REVIEW

File: __test__/integration/debt_token_api/scenarios/owner_of.ts [hunk 1]
```diff
@@ -1,18 +1,60 @@
...
-        description: "the creditor transfers ownership",
-        orders: Orders.ALL_ORDERS,
-        ...Orders.WHO,
-        shouldTransferTo: ACCOUNTS[2].address,
-    },
-];
+const defaults = {
+    orderFilledByCreditorOne: Orders.CREDITOR_ONE_ORDER,
+    orderFilledByCreditorTwo: Orders.CREDITOR_TWO_ORDER,
+    tokenID: (creditorOneTokenID, creditorTwoTokenID, nonexistentTokenID, malFormedTokenID) =>
```
Comment (nadavhollander) [inline __test__/integration/debt_token_api/scenarios/owner_of.ts]: why is `creditorTwoTokenID` an option if neither of the `TokenInjectable` scenario types we've created so far ever leverage it?

File: src/apis/debt_token_api.ts [hunk 2]
```diff
@@ -63,7 +63,7 @@ export class DebtTokenAPI implements ERC721 {
     constructor(web3: Web3, contracts: ContractsAPI) {
         this.web3 = web3;
         this.contracts = contracts;
-        this.assert = new Assertions(web3, this.contracts);
+        this.assert = new Assertions(this.contracts);
```
Comment (nadavhollander) [inline src/apis/debt_token_api.ts]: Assertions takes `web3` as one of its constructor arguments -- please fix.

File: __test__/integration/debt_token_api/scenarios/owner_of.ts (hunk 1 above)
Comment (saturnial) [reply __test__/integration/debt_token_api/scenarios/owner_of.ts]: `TokenInjectable` should handle all four scenarios we care about when it comes to injecting token ids:

1 - You're the owner of the token
2 - You're not the owner of the token
3 - The token id does not exist
4 - The token id is malformed

Not every test suite will leverage all four scenarios though. In our case, calling ownerOf doesn't require you to be the owner of the token, which is why we don't leverage it in our tests.

File: src/apis/debt_token_api.ts (hunk 2 above)
Comment (saturnial) [reply src/apis/debt_token_api.ts]: 🤔

Atom was previously wanting me to remove this. Not sure why...

Fixed.

**Comment by saturnial** [comment]
♻️ @nadavhollander

**Comment by nadavhollander** [comment]
LGTM.  Party on, Wayne.

**Pull request merged**